<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1398</width>
    <height>884</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="start_pushButton">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>750</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Hold</string>
    </property>
   </widget>
   <widget class="QWidget" name="matplotlib_widget" native="true">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>1031</width>
      <height>691</height>
     </rect>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-20</x>
      <y>60</y>
      <width>1041</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="SaveFig_pushButton">
    <property name="geometry">
     <rect>
      <x>740</x>
      <y>750</y>
      <width>100</width>
      <height>30</height>
     </rect>
    </property>
    <property name="text">
     <string>Save Fig</string>
    </property>
   </widget>
   <widget class="QFrame" name="frame">
    <property name="geometry">
     <rect>
      <x>1070</x>
      <y>130</y>
      <width>231</width>
      <height>541</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Raised</enum>
    </property>
    <widget class="QLabel" name="label_2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>160</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Volt</string>
     </property>
    </widget>
    <widget class="QPushButton" name="gridlinescolor_pushButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>230</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Grid Line Color</string>
     </property>
    </widget>
    <widget class="QPushButton" name="bkgndclr_pushButton">
     <property name="geometry">
      <rect>
       <x>70</x>
       <y>290</y>
       <width>100</width>
       <height>30</height>
      </rect>
     </property>
     <property name="text">
      <string>Background Color</string>
     </property>
    </widget>
    <widget class="QSlider" name="timeSlider">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>40</y>
       <width>22</width>
       <height>111</height>
      </rect>
     </property>
     <property name="maximum">
      <number>2</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QSlider" name="timeSlider2">
     <property name="geometry">
      <rect>
       <x>160</x>
       <y>370</y>
       <width>22</width>
       <height>111</height>
      </rect>
     </property>
     <property name="maximum">
      <number>2</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_5">
     <property name="geometry">
      <rect>
       <x>150</x>
       <y>490</y>
       <width>61</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Time</string>
     </property>
    </widget>
    <widget class="QSlider" name="voltSlider2">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>370</y>
       <width>22</width>
       <height>111</height>
      </rect>
     </property>
     <property name="maximum">
      <number>2</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_4">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>490</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Volt</string>
     </property>
    </widget>
    <widget class="QSlider" name="voltSlider">
     <property name="geometry">
      <rect>
       <x>60</x>
       <y>40</y>
       <width>22</width>
       <height>111</height>
      </rect>
     </property>
     <property name="maximum">
      <number>2</number>
     </property>
     <property name="pageStep">
      <number>1</number>
     </property>
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
    </widget>
    <widget class="QLabel" name="label_3">
     <property name="geometry">
      <rect>
       <x>140</x>
       <y>160</y>
       <width>31</width>
       <height>31</height>
      </rect>
     </property>
     <property name="text">
      <string>Time</string>
     </property>
    </widget>
   </widget>
   <zorder>start_pushButton</zorder>
   <zorder>matplotlib_widget</zorder>
   <zorder>line</zorder>
   <zorder>SaveFig_pushButton</zorder>
   <zorder>fontComboBox</zorder>
   <zorder>frame</zorder>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1398</width>
     <height>26</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionAbout">
   <property name="text">
    <string>About</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
